src/pybind/mgr/mgr_module.py

```diff
@@ -40,6 +42,25 @@
     "creating",
     "unknown"]
 
+def is_ipv6_enabled():
+    """
+    Check whether IPv6 is enabled.
```

Comment:
If you copy and paste GPL code, then you should fulfill the license. The copyright of this code is mine and the openmediavault project.

Reply:
ok, sorry for this

Reply:
@kungf please don't copy and paste GPL code in this way as it will render the whole project a derivative of it, unless the author licensed the code under a LGPL 2.1 compatible license.